### PR TITLE
fix: update the resource file for BaseRankingEvaluator

### PR DESCRIPTION
**Pull request by nan-wang**
fix: update the resource file for BaseRankingEvaluator

### CODE REVIEW

**Comment by florian-hoenicke** [comment]
Can we have a small test or adjust an existing test?

**Review by hanxiao — CHANGES_REQUESTED** [review]
im not sure this is a valid change, `tags.id` is not always there unless you fill it. `BaseRankingEvaluator` should not depend on a field that not exist.

**Comment by nan-wang** [comment]
@hanxiao @JoanFM @florian-hoenicke Instead of changing the resource file, I've just added a unit test to show the issue. 

In https://github.com/jina-ai/jina/blob/master/jina/drivers/evaluate.py#L124, we provide the default value for fields as `('tags__id',)`.

However, in the resource file, https://github.com/jina-ai/jina/blob/master/jina/resources/executors.requests.BaseRankingEvaluator.yml#L17, we provide the default value for fields as `['id']`. 

This inconsistency might introduce confusion for the users. I think Han's argument is valid, and therefore suggest to remove the default value in the resource file and change the default value in `RankEvaluateDriver` to `('id', )`

File: jina/drivers/evaluate.py [hunk 1]
```diff
@@ -144,7 +146,7 @@ def extract(self, doc: 'Document'):
         Extract values of the matches from documents with fields as keys.
 
         :param doc: Documents to be extracted.
-        :return: Extracted data.
+        :return: a list of tuples consisted of the values from the fields.
```
Comment (cristianmtr) [inline jina/drivers/evaluate.py:149]:
```suggestion
        :return: a list of tuples consisting of the values from the fields.
```

Comment (nan-wang) [reply jina/drivers/evaluate.py:149]: good catch!

**Review by JoanFM — APPROVED** [review]

**Pull request merged**